php/Blockonomics.php

```diff
@@ -29,7 +32,7 @@ public function get_api_key()
     }
 
 
-    public function new_address($secret, $reset=false)
+    public function new_address($secret, $crypto,$reset=false)
```

Comment:
See a call to this function from line 271 in `test_setup()` Function params changed but the call did not so its not actually doing reset but passing `$crypto` as True and reset defaults to false